src/utils/openai.rs

```diff
@@ -0,0 +1,119 @@
+use reqwest::Error;
+use serde::Deserialize;
+use std::process;
+
+const MODEL_TO_USE: &str = "gpt-3.5-turbo";
```

Comment:
Might make this configurable through a `--model` parameter with this at the default.

Reply:
Added with the `--model` flag. Will still fallback to `gpt-3.5-turbo` if nothing specifically specified.